src/main/java/be/seeseemelk/mockbukkit/ChunkMock.java

```diff
@@ -280,6 +280,13 @@ public boolean contains(@NotNull Biome biome)
 		throw new UnimplementedOperationException();
 	}
 
+	@Override
+	public @NotNull LoadLevel getLoadLevel()
+	{
+		// TODO Auto-generated method stub
+		throw new UnimplementedOperationException();
+	}
+
```

Comment:
Can you please move this to a seperate PR like #764 does for example. Stubbing new Methods shouldn't be done in a PR that adds Implementations